### PR TITLE
avoid initial fitBounds

### PR DESCRIPTION
**Pull request by karussell**
this avoids the jump on load. the jump, when two points are specified, still exists.

TODO: zoom a bit into the world (if world boundary) to avoid seeing some continents twice.

related to #53

### CODE REVIEW

**Comment by easbar** [comment]
Closing here in favor of #77. The initial jump when points are specified remains even in #77. I opened #82 for this.

File: src/api/graphhopper.d.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { LineString } from 'geojson'
 
+// min lat, min lng, max lat, max lng
```
Comment (easbar) [inline src/api/graphhopper.d.ts:3]: I think it is actually min lng, min lat, max lng, max lat, for example world = [-180, -90, 180, 90], but yes, adding such a comment might help :)

Comment (karussell) [reply src/api/graphhopper.d.ts:3]: oh, strange. I thought I did my research properly :/

Comment (karussell) [reply src/api/graphhopper.d.ts:3]: Uh, looks like I had an inconsistent state in my mind ;)

https://github.com/graphhopper/graphhopper-maps/compare/master...test_current_location#diff-516d68fe9c8c9fc0d5cc075d0ba6348d77cee435c4b9eb0bad4c844d9af215b6R22

**Pull request closed without merging**